src/billing/ConsolidatedSpendReport.tsx

```diff
@@ -0,0 +1,427 @@
+        }}
+      >
+        <div role='columnheader' aria-sort={ariaSort(sort, 'billingAccount')} style={{ flex: 1, paddingLeft: '2rem' }}>
+          <HeaderRenderer sort={sort} onSort={() => handleSort('billingAccount')} name='billingAccount' />
+        </div>
+        <div role='columnheader' aria-sort={ariaSort(sort, 'workspaceName')} style={{ flex: 1 }}>
+          <HeaderRenderer sort={sort} onSort={() => handleSort('workspaceName')} name='workspaceName' />
+        </div>
+        <div role='columnheader' aria-sort={ariaSort(sort, 'totalSpend')} style={{ flex: 1 }}>
+          <HeaderRenderer sort={sort} onSort={onSort} name='totalSpend' />
+        </div>
+        <div role='columnheader' aria-sort={ariaSort(sort, 'totalCompute')} style={{ flex: 1 }}>
+          <HeaderRenderer sort={sort} onSort={onSort} name='totalCompute' />
+        </div>
+        <div role='columnheader' aria-sort={ariaSort(sort, 'totalStorage')} style={{ flex: 1 }}>
+          <HeaderRenderer sort={sort} onSort={onSort} name='totalStorage' />
+        </div>
+        {/* <div role='columnheader' aria-sort={ariaSort(sort, 'otherSpend')} style={{ flex: 1 }}>
+          <HeaderRenderer sort={sort} onSort={onSort} name='otherSpend' />
+        </div> */}
+        <div role='columnheader' aria-sort={ariaSort(sort, 'createdBy')} style={{ flex: 1 }}>
+          <HeaderRenderer sort={sort} onSort={onSort} name='createdBy' />
+        </div>
+        <div
+          role='columnheader'
+          aria-sort={ariaSort(sort, 'lastModified')}
+          style={{ flex: `0 0 ${workspaceLastModifiedWidth}px` }}
+        >
+          <HeaderRenderer sort={sort} onSort={onSort} name='lastModified' />
+        </div>
+      </div>
+    );
+  }
+);
+
+interface ConsolidatedSpendWorkspaceCardProps {
+  workspace: GoogleWorkspaceInfo;
+  billingAccountDisplayName: string | undefined;
+  billingProject: BillingProject;
+  billingAccountStatus: false | BillingAccountStatus;
+}
+
+const ConsolidatedSpendWorkspaceCard: React.FC<ConsolidatedSpendWorkspaceCardProps> = memoWithName(
+  'ConsolidatedSpendWorkspaceCard',
+  (props: ConsolidatedSpendWorkspaceCardProps) => {
+    const { workspace, billingAccountDisplayName, billingProject, billingAccountStatus } = props;
+    const { namespace, name, createdBy, lastModified, totalSpend, totalCompute, totalStorage } = workspace;
+    const workspaceCardStyles = {
+      field: {
+        ...Style.noWrapEllipsis,
+        flex: 1,
+        height: '1.20rem',
+        width: `calc(50% - ${workspaceLastModifiedWidth / 2}px)`,
+        paddingRight: '1rem',
+      },
+      row: { display: 'flex', alignItems: 'center', width: '100%', padding: '1rem' },
+    };
+
+    return (
+      <div role='row' style={{ ...Style.cardList.longCardShadowless, padding: 0, flexDirection: 'column' }}>
+        <div style={workspaceCardStyles.row}>
+          <div role='cell' style={workspaceCardStyles.field}>
+            {billingAccountDisplayName ?? '...'}
```

Comment:
It might be nice to link to the billing project from the table as well like what we're doing with the workspace dashboard